src/index.ts

```diff
@@ -129,3 +129,50 @@ export * from './matrix/matrixPQN';
 export * from './matrix/matrixZRescale';
 
 export * from './utils/createSequentialArray';
+
+export type ArrayType = number[] | Float64Array | Float32Array | Uint16Array;
```

Comment:
Please only allow number[] | Float64Array

Reply:
but there is unit16Array used in tests

Reply:
Resolved

Reply:
Fixed

Reply:
@lpatiny 
Unit16Array used in:
https://github.com/mljs/spectra-processing/blob/aa228beb14e321cd8d43eddec89c1ad9923d7561/src/x/__tests__/xSum.test.js#L4
Float32Array used in:
https://github.com/mljs/spectra-processing/blob/aa228beb14e321cd8d43eddec89c1ad9923d7561/src/x/__tests__/xSubtract.test.js#L16

Reply:
Ping @lpatiny